backend/controllers/iepHistory.js

```diff
@@ -1,7 +1,7 @@
 const moment = require('moment')
 
 const getIepHistoryFactory = elite2Api => {
-  const getIepHistory = async (context, offenderNo) => {
+  const getIepHistory = async (context, offenderNo, params) => {
```

Comment:
It would be good to do some validation here. The reason I say this is we've noticed that in production it that the client side validation doesn't seem to kick in.  

This should help minimise the logs across all services.

Reply:
Talked this over in person, form data is not used in the elite2 api calls, so there is no need for validation here.